bin/node/runtime/src/lib.rs

```diff
@@ -1478,6 +1478,7 @@ impl pallet_assets::Config for Runtime {
 	type StringLimit = StringLimit;
 	type Freezer = ();
 	type Extra = ();
+	type CallbackHandle = ();
```

Comment:
(Sorry am late to the party.), was just wondering if `OnCreated = (); OnDestroyed = ()` might make it a bit more similar to other pallet callbacks?

Reply:
My idea was to keep it simple by only having one type but I do like your suggestion. It's more verbose :)